### PR TITLE
remove un-used members

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Paramore.Brighter.MessagingGateway.AWSSQS/SqsMessageConsumer.cs [hunk 1]
```diff
@@ -260,17 +260,6 @@ public bool Requeue(Message message, int delayMilliseconds)
             }
         }
 
-        private string FindTopicArnByName(RoutingKey topicName)
```
Comment (iancooper) [inline src/Paramore.Brighter.MessagingGateway.AWSSQS/SqsMessageConsumer.cs:263]: I'd rather keep this one, for now

File: src/Paramore.Brighter.Outbox.Sqlite/SqliteOutboxSync.cs [hunk 2]
```diff
@@ -41,11 +41,8 @@ namespace Paramore.Brighter.Outbox.Sqlite
     /// </summary>
     public class SqliteOutboxSync : RelationDatabaseOutboxSync<SqliteConnection, SqliteCommand, SqliteDataReader, SqliteParameter>
     {
-        private static readonly ILogger s_logger = ApplicationLogging.CreateLogger<SqliteOutboxSync>();
```
Comment (iancooper) [inline src/Paramore.Brighter.Outbox.Sqlite/SqliteOutboxSync.cs:44]: Again, this forms a standard pattern for us and is likely to be needed - so let's leave it

File: src/Paramore.Brighter.Outbox.Sqlite/SqliteOutboxSync.cs [hunk 3]
```diff
@@ -41,11 +41,8 @@ namespace Paramore.Brighter.Outbox.Sqlite
...
-
         private const int SqliteDuplicateKeyError = 1555;
         private const int SqliteUniqueKeyError = 19;
-        private readonly SqliteConfiguration _configuration;
```
Comment (iancooper) [inline src/Paramore.Brighter.Outbox.Sqlite/SqliteOutboxSync.cs:48]: Again, with this

**Review by iancooper — CHANGES_REQUESTED** [review]
I'd suggest we don't clean these up. This kind of clean up has value, but there is a lot here about expectations across a range of code.

**Comment by iancooper** [comment]
@SimonCropp I might close this one, I don't think we want to make these changes

**Comment by SimonCropp** [comment]
no worries. now i am aware of the pattern 😀

**Pull request closed without merging**